woocommerce/woocommerce-admin
# Add woo_country_code param to the assignment fetch request in ExPlat

Fixes #7527

This PR adds `woo_country_code` param to the assignment fetch request in ExPlat.

### Detailed test instructions:

1. Clear your local storage by going to Inspector -> Application -> Storage -> Local Storage (Chrome)
2. Navigate to WooCommerce -> Home
3. Open `Network` tab and search `assignment`. Click one of the requests and confirm the request has `woo_country_code`

no changelog
